anypoint-signin-aware.js

```diff
@@ -13,6 +13,15 @@ the License.
 */
```

Comment:
Please remove the copyright/license noise and replace with a very clear statement of purpose for this module.
Also the naming here is inconsistent and thus confusing. The file name does not match the exported object name.